bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -250,16 +249,16 @@ spec:
                 - name: RELEASE_VERSION
                   value: v0.0.1
                 - name: HW_EVENT_PROXY_IMAGE
-                  value: quay.io/redhat-cne/hw-event-proxy:4.10
+                  value: quay.io/openshift/origin-baremetal-hardware-event-proxy:4.10
                 - name: KUBE_RBAC_PROXY_IMAGE
                   value: quay.io/openshift/origin-kube-rbac-proxy:4.10
                 - name: CLOUD_EVENT_PROXY_IMAGE
-                  value: quay.io/redhat-cne/cloud-event-proxy:4.10
+                  value: quay.io/openshift/origin-cloud-event-proxy:4.10
                 - name: WATCH_NAMESPACE
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
-                image: quay.io/redhat-cne/hw-event-proxy-operator:0.0.1
+                image: quay.io/openshift/origin-baremetal-hardware-event-proxy-operator:latest
```

Comment:
this should 4.11 too and have another manifest PR just for release 4.10.
4.11 changes will go to master